crates/errors/src/error_codes/META0012.md

```diff
@@ -0,0 +1,9 @@
+## META0012
+
+Trying to register a service endpoint whose supported service protocol versions is incompatible with the server. This indicates that you have to update your server to make it work together with the deployed SDK.
+
+Suggestions:
+
+* Check the compatibility matrix between SDK and server versions
+  * Either deploy a server version which is compatible with your SDK
+  * Or use an SDK version which is compatible with your server
```

Comment:
This doesn't seem to be a good suggestion to me. If I already developed the service against the newest version of the SDK, there is a good chance I can't easily downgrade to the old SDK version. I would phrase this a bit differently...

Reply:
It is just a suggestion. If this is not possible for a user then the user needs to upgrade the server. There is not much else the person can do.